packages/cra-template-cohtm/README.md

```diff
@@ -0,0 +1,10 @@
+# cra-template-cohtml
+
+This is the official base template for [Coherent Labs' Gameface](https://coherent-labs.com/products/coherent-gameface/).
+It provides the foundation required for create-react-app to work with live reload in Gameface.
+
+To create a react application with this template run:
+
+`create-react-app my-test-app --scripts-version cohtml-react-scripts --template cra-template-cohtml`
```

Comment:
In the [document in cohtml](https://github.com/CoherentLabs/cohtml/pull/2233/files#diff-d89e998d7f5da0434774974a16febb6b479cf796e163259ce33a7aea5cb58b82) this part is different there -> `--scripts-version react-scripts-cohtml`.
Fix with the correct one.

It seems that `react-scripts-cohtml` is the true one.